### DISCUSSION TITLE
GitHub Copilot corrupts/crashes VS 2022 latest version

### DISCUSSION THREAD

**Discussion by Spazjuaner** [community/community#18634] [Copilot Conversations]
Installing a fresh version of the latest VS 2022, then installing GitHub Copilot from extensions causes the whole thing to crash and throw errors. Almost like it corrupts it
![trying to close](https://user-images.githubusercontent.com/89706931/173737780-17814c60-67f5-4d51-9fe9-4eda84dfda4d.png)
 then trying to start a new project no installs seems to be there anymore.
![all instals and tools gone](https://user-images.githubusercontent.com/89706931/173737885-db72c2e3-6643-40b5-9793-9222c84c5c63.png)

Opening a old cpp project
![cppcant open](https://user-images.githubusercontent.com/89706931/173738037-ac2678a8-61c1-412e-9b8b-4b3453bdd397.png)

"An error occurred in 'Miscellaneous Files' while attempting to open 'ConsoleApplication2.cpp'
There is no editor available for 'C:\Users\TechFast Australia\source\repos\ConsoleApplication2\ConsoleApplication2\ConsoleApplication2.cpp'.

Make sure the application for the file type (.cpp) is installed."

Have used https://github.com/microsoft/VisualStudioUninstaller/releases/tag/v1.4
Ive reinstalled a dozen times now and hit a brick wall on things to try.
Any assistance would be greatly appreciated.

Thanks

**Comment by Spazjuaner** [comment]
[ActivityLog.zip](https://github.com/github-community/community/files/8923072/ActivityLog.zip)
forgot to add the xml file showing the errors, I cant seem to figure them out.
=(

**Comment by Spazjuaner** [comment]
Seems to be Vs2022 VSIX updater, any install or update completely corrupts VS and I have to repair.

**Reply by kzu** [reply]
Looks like your VS install has some issues. Also, you're using a pretty old VS: `Microsoft Visual Studio 2022 version: 17.0.32602.120`.

Please update to the latest stable release (should be 17.3 at least). Copilot is not supported under 17.1 or prior versions).

Thanks
